vuepress/docs/next/tutorials/solution/install-standard-demo.md

```diff
@@ -62,120 +89,142 @@ ent ecr deploy --repo="https://github.com/entando-samples/standard-demo-manage-u
 ent ecr deploy --repo="https://github.com/entando-samples/standard-demo-content-bundle.git"
 ```
 
-2. Log into your App Builder instance.
+2. Log in to your App Builder instance.
+
+3. Select `Hub` from the menu on the left to view the bundles in your Local Hub.
 
-3. Select `Hub` from the menu on the left. Your bundles will be visible in the repository as shown in the screenshot below.
-   ![Repository.png](./images/Repository.png)
+4. Select `Deploy` and `Install` for each bundle:
 
-4. Select `Deployed` and `Install` for each bundle, where order of installation is important. The `standard-demo-content-bundle` will need to be installed last, as it relies on MFEs from the other bundles to set up each of the pages. 
-   ![Installed.png](./images/Installed.png)
+     1. Click on the bundle entry
+     2. In the pop-up box, click "Deploy"
+     3. Click on the "Install" button that appears in this same pop-up box once the bundle deploys
+     4. Exit out of the pop-up box
+     5. Repeat steps 1-4 for each bundle
 
-Each installation can take several minutes while the application downloads the Linux images for the microservices and installs the related assets. The `standard-demo-banking-bundle` and `standard-demo-customer-bundle` include microservices that require the initialization of containers and will take longer to install.
+     ![sd-local-hub2.png](./sd-images/sd-local-hub2.png) 
 
-In the unlikely event you encounter conflicts during an initial installation, you will be presented with an Installation Plan like the one shown below. Select `Update All` in the upper right after making your selections.
-   ![InstallPlan.png](./images/InstallPlan.png)
+     **Order of installation is important.** The `standard-demo-content-bundle` must be installed last, as it relies on MFEs from the other bundles to set up each of the pages.
```

Comment:
Minor but as a full sentence, think you need 'the' for first sentence>>The order of installation is important.